applications/asset_tracker_v2/src/events/ui_module_event.h

```diff
@@ -20,27 +20,45 @@
 extern "C" {
 #endif
 
-/** @brief UI event types submitted by UI module. */
+/** @brief UI event types submitted by the UI module. */
 enum ui_module_event_type {
+	/** Button has been pressed.
+	 *  Payload is of type @ref ui_module_data (ui).
+	 */
 	UI_EVT_BUTTON_DATA_READY,
-	UI_EVT_SHUTDOWN_READY,
-	UI_EVT_ERROR
+
+	/** The sensor module has performed all procedures to prepare for
+	 *  a shutdown of the system. The event carries the ID (id) of the module.
+	 */
+	SENSOR_EVT_SHUTDOWN_READY,
+
+	/** An irrecoverable error has occurred in the cloud module. Error details are
+	 *  attached in the event structure.
+	 */
+	SENSOR_EVT_ERROR
```

Comment:
Replacing `UI` prefix with `SENSOR` seems like a unintended mistake. Due to this change CI tests are failing.